src/main/drivers/bus_i2c_stm32f10x.c

```diff
@@ -18,18 +18,21 @@
 #include <stdbool.h>
 #include <stdint.h>
 #include <stdlib.h>
+#include <string.h>
 
 #include <platform.h>
 
 #include "drivers/io.h"
 #include "drivers/time.h"
 
-#include "drivers/bus_i2c.h"
 #include "drivers/nvic.h"
-#include "io_impl.h"
-#include "rcc.h"
+#include "drivers/io_impl.h"
```

Comment:
`_impl.h` is meant to contain declaration common to files implementing driver - internals that should not be normally exposed. Normal file should not need to include it ...